x/dualstaking/keeper/delegate.go

```diff
@@ -424,7 +422,7 @@ func (k Keeper) getUnbondHoldBlocks(ctx sdk.Context, chainID string) uint64 {
 	// NOT REACHED
 }
 
-func (k Keeper) getMinStake(ctx sdk.Context, chainID string) sdk.Coin {
+func (k Keeper) GetMinStake(ctx sdk.Context, chainID string) sdk.Coin {
```

Comment:
why is this public?

Reply:
Because I'm using this function in `x/pairing/keeper/msg_server_unfreeze.go`